### ISSUE TITLE
BasePath with absolute path in css 

### ISSUE THREAD

**Issue by standy** [justim/postcss-svg-fallback#17]
I'm trying to convert

```
.logo {
    background: url(/img/logo.svg);
    background-size: 100px 40px;
}
```

And getting error `Could not find "/img/logo.svg" at "D:\img\logo.svg"`

Changing `basePath` and `dest` does not affect to error message. I suppose that happens becouse of using `path.resolve()` which is same as using `cd` commands in shell.
I think right way to convert css path to basePath is `path.join()`

Could you approve it?

**Comment by justim** [comment]
By using `path.join` the resulting path is not necessarily an absolute path, which _might_ cause problems for some people and/or in future releases that do something with the current working directory. So I think we need to find another way to locate your file, maybe by stripping the leading `/` in your background image path?

**Comment by standy** [comment]
You can always get absolute path using current working directory, same as `path.resolve` do. Or just use `path.resolve(path.join(...))`  
Stripping the `/` will work in my case, but what if there is absolute and relative path is same css — they should be threated in a different way

**Comment by justim** [comment]
Completely forgot about pushing the change I made in 99c133d. It is published on npm as version 1.3.0. Hope this solves your problem :+1:
